skills/bear-resource-generator/SKILL.md

```
@@ -0,0 +1,608 @@
+   ```
+
+2. Run tests:
+   ```bash
+   cd app && composer test
+   ```
+
+3. Fix coding standards:
+   ```bash
+   cd app && composer cs-fix
+   ```
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)